builder/tests/08-unrecognized-attribute.stderr

```diff
@@ -0,0 +1,4 @@
+error: missing or invalid `each` parameter in builder
+   |
+19 | #[derive(Builder)]
+   |          ^^^^^^^
```

Comment:
It would be nicer if this error would appear directly on the line containing the bad attribute -- something like:

```console
error: missing or invalid `each` parameter in builder

    #[builder(eac = "arg")]
    ^^^^^^^^^^^^^^^^^^^^^^^
```
or even:

```console
error: unrecognized builder attribute `eac`

    #[builder(eac = "arg")]
              ^^^
```

Reply:
Is it possible to get the line verbatim, and the line number? I tried `quote! { attr }` but it adds spaces that are not in the code.

Reply:
The compiler will take care of printing out the line of code. You will want to use [syn::Error::new_spanned](https://docs.rs/syn/0.15/syn/struct.Error.html#method.new_spanned) to put the right span on the error, followed by [syn::Error::to_compile_error](https://docs.rs/syn/0.15/syn/struct.Error.html#method.to_compile_error).